pom.xml

```diff
@@ -47,6 +47,9 @@
 
         <!-- Properties to consolidate slf4j versions -->
         <slf4j-version>1.7.32</slf4j-version>
+        <azure-core-version>1.12.0</azure-core-version>
+        <lombok-version>1.18.8</lombok-version>
+        <azure-storage-blob-version>12.8.0</azure-storage-blob-version>
```

Comment:
We'll have to re-evaluate the recent upgrade we did on azure-core and azure-storage-blob since they break our existing Android support. This PR reverts the upgrade to them made by https://github.com/Azure/azure-iot-sdk-java/commit/5a590f59422529e09d56c0f9a44f2518c5acffd5